modules/openapi-generator/src/test/java/org/openapitools/codegen/typescript/TypeScriptAxiosClientCodegenTestTest.java

```diff
@@ -0,0 +1,81 @@
+package org.openapitools.codegen.typescript;
+
+import org.openapitools.codegen.CodegenConstants;
+import org.openapitools.codegen.languages.TypeScriptAxiosClientCodegen;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class TypeScriptAxiosClientCodegenTestTest {
+
+    TypeScriptAxiosClientCodegen codegen = new TypeScriptAxiosClientCodegen();
+
+    @Test
+    public void testToEnumVarNameOriginalNamingType() {
+        codegen.additionalProperties().put(CodegenConstants.ENUM_PROPERTY_NAMING, CodegenConstants.ENUM_PROPERTY_NAMING_TYPE.original.name());
+        codegen.processOpts();
+        assertEquals(codegen.toEnumVarName("SCIENCE", "string"), "SCIENCE");
+        assertEquals(codegen.toEnumVarName("SCIENCE_FICTION", "string"), "SCIENCE_FICTION");
```

Comment:
I stumbled upon this while bumping openapi-generator version. Is there a bug in this `original` implementation (and this test case)? As the issue stated:

> For example, original enum values:
> 
> ```
>       "BookType": {
>         "type": "string",
>         "description": "Book Type",
>         "example": "SCIENCE",
>         "enum": [
>           "SCIENCE",
>           "SCIENCE_FICTION"
>         ]
>       }
> ```
> Expected enum values:
> 
> ```
> export enum BookType {
>   Science = 'SCIENCE',
>   ScienceFiction = 'SCIENCE_FICTION'
> }
> ```
> Actual enum values:
> ```
> export enum BookType {
>   SCIENCE = 'SCIENCE',
>   SCIENCEFICTION = 'SCIENCE_FICTION'
> }
> ```

So I suppose when using `original` it should generate value without underscores and this test should be asserting value being `SCIENCEFICTION` instead of underscored `SCIENCE_FICTION`?